README.md

```diff
@@ -4,3 +4,9 @@ This repository contains the `genesis.json` for Celestia testnets.
 
 Please refer to the Celestia Docs for guides on getting started running
 your own node [here](https://docs.celestia.org)
+
+## Networks
+
+| Network | celestia-node                                                                      | celestia-app                                                                        | celestia-core                                                                                      | cosmos-sdk                                                                                      |
+| ------- | ---------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------- |
+| Arabica | [v0.5.0-rc5](https://github.com/celestiaorg/celestia-node/releases/tag/v0.5.0-rc5) | [v0.10.0-rc1](https://github.com/celestiaorg/celestia-app/releases/tag/v0.10.0-rc1) | [v1.5.0-tm-v0.34.20](https://github.com/celestiaorg/celestia-core/releases/tag/v1.5.0-tm-v0.34.20) | [v1.4.0-sdk-v0.46.0](https://github.com/celestiaorg/cosmos-sdk/releases/tag/v1.4.0-sdk-v0.46.0) |
```

Comment:
[question for reviewers] where can I find what versions of our software are running on Mamaki?